plugins/openshift/commands/api-review.md

```
@@ -0,0 +1,232 @@
+---
+description: Run strict OpenShift API review workflow for PR changes or local changes
+argument-hint: "[pr_url]"
+---
+
+## Name
+openshift:api-review
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)